oauth2client/client.py

```diff
@@ -279,8 +280,7 @@ def new_from_json(cls, s):
       An instance of the subclass of Credentials that was serialized with
       to_json().
     """
-    if isinstance(s, bytes):
-      s = s.decode('utf-8')
+    s = _from_bytes(s)
```

Comment:
The old code was using `s = s.decode('utf-8')`, so I was just being mindless. Will change to `s_bytes` (and use a better varname than `s`).

New commit or fold into original?